backup_tests/xbstream_fifo_test.sh

```diff
@@ -54,9 +132,35 @@ stop_server() {
 }
 
 init_datadir() {
+  local keyring_type=$1
   if [ -d $DATADIR ]; then
     rm -rf $DATADIR
   fi
+
+  if [ "$keyring_type" = "keyring_kmip" ]; then
+    echo "Keyring type is KMIP. Taking KMIP-specific action..."
+
+    echo '{
+      "components": "file://component_keyring_kmip"
+    }' > "$PS_DIR/bin/mysqld.my"
+
+    setup_kmip
+
+  elif [ "$keyring_type" = "keyring_file" ]; then
+    echo "Keyring type is file. Taking file-based action..."
+
+    echo '{
+      "components": "file://component_keyring_file"
+    }' > "$PS_DIR/bin/mysqld.my"
+
+    cat > "$PS_DIR/component_keyring_file.cnf" <<-EOFL
```

Comment:
global config files must be inside PS_DIR/lib/plugin

Reply:
good catch, fixed it. thanks